scripts/cert_gen.sh

```diff
@@ -0,0 +1,402 @@
+#!/bin/bash
+
+## Copyright (c) Microsoft. All rights reserved.
+## Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+###############################################################################
+# This script demonstrates creating X.509 certificates for an Azure IoT Hub
+# CA Cert deployment.
+# This script is used to create the python build environment
+# on which our e2e tests run.
+# Currently the root cert will have a duration of 1 year.
+# All other certificates created from this root certificate
+# gets deleted after the tests.
+# NOTE : In case of 401 error during running DPS E2E tests for certificates
+# the certificates produced by this has expired. And the build environment
+# needs to run again.
+#
+# These certs MUST NOT be used in production.  It is expected that production
+# certificates will be created using a company's proper secure signing process.
+# These certs are intended only to help demonstrate and prototype CA certs.
+###############################################################################
+
```

Comment:
if you put "set -e" then it will return immediately with the error code so you can call this from another script and on failure it will be able to catch it.